editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -1034,7 +1034,7 @@ void SpriteFramesEditor::_animation_name_edited() {
 	edited->set_text(0, name);
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
-	undo_redo->create_action(TTR("Rename Animation"), UndoRedo::MERGE_DISABLE, frames.ptr());
+	undo_redo->create_action(TTR("Rename Animation"), UndoRedo::MERGE_DISABLE);
```

Comment:
Does it really fix something?
It should have the same effect.

Reply:
ah, yes it really doesn't have an effect, reverted...